lsif-java/src/main/scala/com/sourcegraph/lsif_java/buildtools/GradleJavaCompiler.scala

```diff
@@ -98,6 +98,29 @@ case class GradleJavaCompiler(languageVersion: String, javacPath: Path) {
       )
       .toFile
       .setExecutable(true)
+
+    // for compileKotlin when using jvm toolchains.
+    // JDK 9+ have jrt-fs.jar in $JDK_HOME/lib
+    val libPath = dir.resolve("lib")
+    val javacLibPath = javacPath.getParent.getParent.resolve("lib")
+    Files
+      .walk(javacLibPath)
+      .forEach(source => {
+        val destination = libPath.resolve(javacLibPath.relativize(source))
+        Files.copy(source, destination)
+      })
+
+    // JDK 8 has rt.jar in $JDK_HOME/jre/lib
+    if (languageVersion == "8") {
+      val jrePath = dir.resolve("jre")
+      val javacJrePath = javacPath.getParent.getParent.resolve("jre")
+      Files
+        .walk(javacJrePath)
+        .forEach(source => {
+          val destination = jrePath.resolve(javacJrePath.relativize(source))
+          Files.copy(source, destination)
+        })
+    }
```

Comment:
1. Is it intentional that for languageVersion == 8, we will perform two sets of copies? If not, should it be an`if lang == 8 then do X else do Y` instead of `do Y; if lang == 8 do X?
2. The comment says "JDK 8 has rt.jar in $JDK_HOME/jre/lib" but there is no `lib` in the code below it... I don't understand that. Is that intentional?
3. There is a bunch of code duplication; would be nice to factor it out:
    * either into a small local function if the answer to 1 is Yes.
    * using `if else` should get rid of the duplication if the answer to 1 is No.

Reply:
1. JDK 9+ dont have the `/jre` folder anymore, everything was consolidated (and restructured) into only `/lib`
2. Everything from `/jre` is copied, including `/jre/lib/*`. I can probably just copy stuff from `/jre/lib` though, and ignore `/jre/bin`

Reply:
> JDK 9+ dont have the /jre folder anymore, everything was consolidated (and restructured) into only /lib

👍 . FWIW, I think it may be helpful to explain some of this in the comment. It seems a bit cryptic right now, since it's talking about two different jars in the two different ones.

Perhaps something like:

```
// In JDK 8, rt.jar in $JDK_HOME/jre/lib and <blah> is in $JDK_HOME/lib
// In JDK 9+, both of these are consolidated under $JDK_HOME/lib
```

> I can probably just copy stuff from /jre/lib though, and ignore /jre/bin

That seems like a good idea.